_posts/2017-01-04-OSS-FAQ.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Open Source FAQ for Engineers"
+date: 2017-01-04 12:17
+author: [orta, yayoi]
+categories: [oss, culture]
+series: Open Source by Default
+---
+
+We have a lot of Open Source code. For engineers without considerable experience in the open source realm, understanding some of the copyright issues around copyrights and code ownership can be tricky. I've been working with [dB](http://artsy.github.io/author/db/), our CTO and [Yayoi Shionoiri](http://www.theartgorgeous.com/yayoi-shionoiri/) (our Senior Counsel) on documenting how this works.
```

Comment:
dB. is spelled with a period :) Also maybe flip "our CTO dB., and our senior counsel Yayoi...", I don't think senior counsel needs to be capitalized but I could be wrong.